lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -25,7 +25,7 @@ def initialize(expected_error_or_message, expected_message, &block)
           when nil, UndefinedValue
```

Comment:
This usage of UndefinedValue confuses me. It was supposed be the default value in the method signature.

Reply:
That looks like either another bug or dead code at this point, good eye! I'll go spelunking in a moment.

Reply:
Ah, it's not _wrong_, just referenced from afar: https://github.com/rspec/rspec-expectations/blob/main/lib/rspec/matchers.rb#L773

I think this will confuse future-us less with an explanatory comment, I'll add that here.

Reply:
Right, now I recall it. Good find.